tests/instancetype_test.go

```diff
@@ -320,6 +324,10 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			}
 			preference.Spec.PreferredTerminationGracePeriodSeconds = pointer.Int64(15)
 			preference.Spec.PreferredSubdomain = pointer.String("non-existent-subdomain")
+			preference.Spec.Annotations = map[string]string{
+				"preferred-annotation-1": "1",
```

Comment:
Can you add one of the required annotations here with a different value and assert that the value from the instance type is used and not the preference?

Can you also populate the `VirtualMachine` with one of the preferred annotations and again assert that the original value from the `VirtualMachine` isn't overwritten by the preference in the `VirtualMachineInstance`.

Reply:
@opokornyy Can you have a look at the second test case mentioned again?

Reply:
Done